File: java/program/TV.java
public class TV extends ProdukAudioVisual {
    private String ukuran;
    private String resolusi;
    private String jenisPanel;

    public TV(String idProduk, String namaProduk, String merk, double harga, String garansi, String tipe, String ukuran, String resolusi, String jenisPanel) {
        super(idProduk, namaProduk, merk, harga, garansi, tipe);
        this.ukuran = ukuran;
        this.resolusi = resolusi;
        this.jenisPanel = jenisPanel;
    }

    // Getters
    public String getUkuran() { return ukuran; }
    public String getResolusi() { return resolusi; }
    public String getJenisPanel() { return jenisPanel; }

    // Setters
    public void setUkuran(String ukuran) { this.ukuran = ukuran; }
    public void setResolusi(String resolusi) { this.resolusi = resolusi; }
    public void setJenisPanel(String jenisPanel) { this.jenisPanel = jenisPanel; }
}

File: cpp/program/TV.cpp
#include "ProdukAudioVisual.cpp" 

class TV : public ProdukAudioVisual {
private:
    std::string ukuran;
    std::string resolusi;
    std::string jenisPanel;

public:
    // Constructor
    TV(const std::string& idProduk, const std::string& namaProduk, const std::string& merk,
       double harga, const std::string& garansi, const std::string& tipe,
       const std::string& ukuran, const std::string& resolusi, const std::string& jenisPanel)
        : ProdukAudioVisual(idProduk, namaProduk, merk, harga, garansi, tipe),
          ukuran(ukuran), resolusi(resolusi), jenisPanel(jenisPanel) {}

    // Getters
    std::string getUkuran() const { return ukuran; }
    std::string getResolusi() const { return resolusi; }
    std::string getJenisPanel() const { return jenisPanel; }

    // Setters
    void setUkuran(const std::string& ukuran) { this->ukuran = ukuran; }
    void setResolusi(const std::string& resolusi) { this->resolusi = resolusi; }
    void setJenisPanel(const std::string& jenisPanel) { this->jenisPanel = jenisPanel; }
};